src/fallback_html/405.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Verse: Method Not Allowed</title>
```

Comment:
Let's add 405 to the title too, that way at least one of the titles has the "correct" name

Reply:
Done